qemu/tests/virtio_driver_sign_check.py

```diff
@@ -57,6 +57,8 @@ def run(test, params, env):
         file_type = [".cat", ".sys", ".inf", "Wdf"]
         tested_list = []
         viowin_letter, path = get_driver_file_path(session, params)
+        if not viowin_letter:
+            test.fail("Cannot find the virtio_win drive letter.")
```

Comment:
should be test.error()
And better to handle this error in vt as well, otherwise lots of tp-qemu jobs need to handle it.

Reply:
thanks lijin, I updated code to test.error,  and I  want to keep the origin design in avocado-vt, return correct drive letter or None directly, user can report customize message through judge this return value, thanks again~